src/entry_points/gentest.py

```diff
@@ -0,0 +1,303 @@
+"""
+Define an entry point wrapper for test generator.
+"""
+
+import json
+import os
+import sys
+from dataclasses import asdict, dataclass
+from typing import Dict, List, Union
+
+import requests
+
+from ethereum_test_tools import Account, Address, Transaction
+
+
+def main():  # noqa: D103
+    if len(sys.argv) != 3:
+        print_help()
+        sys.exit(1)
+    if sys.argv[1] in ["-h", "--help"]:
+        print_help()
+        sys.exit(0)
+    make_test(sys.argv)
+
+
+def print_help():  # noqa: D103
+    print("Extracts transaction and required state from the mainnet to make a BC test out of it")
+    print("Usage: gentest <tx_hash> <file.py>")
+
+
+if __name__ == "__main__":
+    main()
+
+
+def make_test(args):  # noqa: D103
+    transaction_hash = args[1]
+    output_file = args[2]
+    print("Load configs...")
+    config = PyspecConfig(os.path.expanduser("~/.pyspec/config"))
+    request = RequestManager(config.remote_nodes[0])
+
+    print("Perform tx request: eth_get_transaction_by_hash(" + f"{transaction_hash}" + ")")
+    tr = request.eth_get_transaction_by_hash(transaction_hash)
+
+    print("Perform debug_trace_call")
+    state = request.debug_trace_call(tr)
+
+    print("Generate py test >> " + output_file)
+    test = make_test_template(tr, state)
+    with open(output_file, "w") as file:
+        file.write(test)
+
+    print(test)
+
+
+def make_test_template(
+    tr: "RequestManager.RemoteTransaction", state: Dict[Address, Account]
+) -> str:
+    """
+    Prepare the .py file template
+    """
+    test = PYTEST_TEMPLATE
+    test = test.replace(
+        "$HEADLINE_COMMENT",
+        "gentest autogenerated test with debug_traceCall of tx.hash " + tr.tr_hash,
+    )
+    test = test.replace("$TEST_NAME", "test_transaction_" + tr.tr_hash[2:])
+    test = test.replace(
+        "$TEST_COMMENT", "gentest autogenerated test for tx.hash " + tr.tr_hash[2:]
+    )
+
+    # Print a nice .py storage pre
+    pad = "            "
+    state_str = ""
+    for address, account in state.items():
+        if isinstance(account, dict):
+            account_obj = Account(**account)
+            state_str += '        "' + str(address) + '": Account(\n'
+            state_str += pad + "balance=" + str(account_obj.balance) + ",\n"
```

Comment:
```suggestion
            state_str += f"{pad}balance={account_obj.balance},\n"
```
Same for the rest of the lines.